docs/environment.yml

```diff
@@ -0,0 +1,29 @@
+# conda env create -f environment.yml
+name: clisops-docs
+channels:
+- conda-forge
+- defaults
+dependencies:
+- sphinx
+- nbsphinx
+- ipython
+- pandoc
+- pip
+- numpy>=1.16
+- xarray>=0.15
+- pandas>=1.0.3
+- cftime>=1.0.4
+- netCDF4>=1.4
+- poppler>=0.67
+- fiona>=1.8
+- geojson>=2.5.0
+- shapely>=1.6
+- geopandas>=0.7
+- rtree>=0.9
+- dask>=2.6.0
+- bottleneck>=1.3.1,<1.4
+- pyproj>=2.5
+- udunits2>=2.2
+- python-dateutil>=2.8.1
+- pip:
```

Comment:
I would use the conda environment only for conda packages. We can use the `requirements.txt` to list the packages only available on pypi. By this we can generate a conda spec file for a release.

Reply:
Ah ... this is the docs environment ... it would not matter much here.